Documentation/ApiOverview/Routing/AdvancedRoutingConfiguration.rst

```diff
@@ -181,7 +181,8 @@ The configuration looks like this:
           category_id: 'category'
 
 :yaml:`routePath` defines the static keyword and the placeholders.
-:yaml:`requirements.category_id` is a unique integer of the category.
+:yaml:`requirements.category_id` is a unique integer of the category. 
+Without `_arguments` the parameter must have been 'category_id' as well .
```

Comment:
The example writes `index.php?id=13&category=241&tag=Benni`. So, without using `_arguments` the variable has to exactly match the argument name from URI. Here would be the new example code:

```
routeEnhancers:
  CategoryListing:
    type: Simple
    limitToPages: [1]
    routePath: '/show-by-category/{category}'
    requirements:
      category: '[0-9]{1,3}'
```